src/plugins/themes.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const requireg = require("requireg");
+const colors = require("colors/safe");
+const path = require("path");
+const themes = new Map();
+
+module.exports = {
+	get: get,
+	fileName: fileName
+};
+
+fs.readdir("client/themes/", (err, dirThemes) => {
+	if (err) {
+		return;
+	}
+	dirThemes
+		.filter((theme) => theme.endsWith(".css"))
+		.map(makeLocalThemeObject)
+		.forEach((theme) => themes.set(theme.name, theme));
```

Comment:
Is this for backwards compatibility only? Mind adding a comment above this block?

Reply:
It has been a while since I've looked through the code... but yeah, I think it's just for the existing built in themes.

Reply:
I've renamed some stuff here, hopefully better?